modules/jvm-trace/data/jvm_notebook.json

```diff
@@ -0,0 +1,126 @@
+{
+  "cells": [
+    {
+      "content": "# Setup\nInstall required packages and configure pods for testing.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "## Setup Kubernetes\nSetup kubernetes objects for the test.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app =~ \"shoreline\" | limit=1 | `kubectl apply -f  \"/tmp/jvm_test.yaml\"`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "If you ran into permission problem, run the apply command (kubectl apply -f \"/tmp/jvm_test.yaml\") with a user having sufficient permission. Content of the k8s manifest file:",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app =~ \"shoreline\" | limit=1 | `cat  \"/tmp/jvm_test.yaml\"`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "Verify that jvm-test pods have been created.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app = \"jvm-test\"",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "# JVM Trace",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "## Check Memory Usage\n\n",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "Start JVM process",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pod | app='jvm-test' | `cd /tmp; javac EatResources.java && java EatResources &`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
```

Comment:
Should probably duplicate the `... | testjvm_jvm_mem_usage('EatResources')` command and comment here, so the user can see and compare "before" and "after" values.